pkg/project/selector.go

```diff
@@ -0,0 +1,15 @@
+package project
+
+import (
+	applabels "github.com/openshift/odo/pkg/application/labels"
+	"github.com/openshift/odo/pkg/util"
+)
+
+// GetSelector returns a selector to filter resource under the current project created by odo
+func GetSelector() string {
```

Comment:
I feel the file should be called labels.go like in case of app.

Reply:
Ack